File: trunk/quests/WorkInProgress/Omens of War Cleric Epic/Skyfire/Zordak_Ragefire.lua
-- 1.0 Cleric Epic // Water Sprinkler of Nem Ankh // -- Drogerin

function event_death_complete(e)
		eq.spawn2(91090,0,0,e.self:GetX(),e.self:GetY(),e.self:GetZ(),e.self:GetHeading()); -- Spawns Zordakalicus Ragefire
end

File: skyfire/Zordak_Ragefire.pl
# EPIC CLERIC -skyfire-

sub EVENT_DEATH_COMPLETE {
    my $x = $npc->GetX();
    my $y = $npc->GetY();
    my $z = $npc->GetZ();
    my $h = $npc->GetHeading();
  quest::spawn2(91090,0,0,$x,$y,$z,$h); # Zordakalicus_Ragefire
}


#End of File, Zone:skyfire  NPC:32038 -- Zordak Ragefire
